WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardView.swift

```diff
@@ -40,6 +54,44 @@ final class BlazeCampaignDashboardViewHostingController: SelfSizingHostingContro
 }
 
 private extension BlazeCampaignDashboardViewHostingController {
+    private func createBlazeNavigationController() {
+        blazeNavigationController = UINavigationController(rootViewController: createProductSelectorViewController())
+    }
+
+    private func createProductSelectorViewController() -> ProductSelectorViewController {
```

Comment:
Here we need to supply both `onProductSelectionStateChanged` and `onVariationSelectionStateChanged` because they're called separately when a product is tapped, depending if a product is simple/bundle or variation product.

Reply:
onVariationSelectionStateChanged is not needed after all.